scripts/change_url

```diff
@@ -110,38 +110,35 @@ fi
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
-ynh_print_info --message="Modifying a config file..."
+ynh_script_progression --message="Modifying a config file..."
 
-ynh_replace_string --match_string="$old_domain" --replace_string="$new_domain" --target_file="$final_path/$app/.env"
-ynh_replace_string --match_string="$old_domain" --replace_string="$new_domain" --target_file="$final_path/$app/config/prod.secret.exs"
+config="$final_path/$app/config/prod.secret.exs"
+ynh_backup_if_checksum_is_different --file="$config"
 
-#=================================================
-# STORE THE CONFIG FILE CHECKSUM
-#=================================================
-ynh_print_info --message="Storing the config file checksum..."
+ynh_replace_string --match_string="$old_domain" --replace_string="$new_domain" --target_file="$config"
```

Comment:
Ah I realize this is not going to be enough anymore, we also need to tweak the LDAP user that's used for email authentication :/

Reply:
That has to be tested but it's `__YNH_USER__` that is used for authentication https://github.com/YunoHost-Apps/mobilizon_ynh/blob/1fdf878a8d4915bfe01c47c8eb1adbbf2fde3fb6/scripts/install#L170

So maybe, just tweaking the main mail address of the notification user is enough...